portfolio/src/main/webapp/script.js

```diff
@@ -17,10 +17,10 @@
  */
```

Comment:
The comment is for function addRandomGreeting().

Reply:
fixed